examples/local-state-with-normalize-props-proxy/cosmos.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  componentPaths: ['components'],
+  // TODO: use package for normalize proxy
+  proxies: ['../../packages/react-cosmos-normalize-props-proxy/src']
```

Comment:
No need for a TODO since examples are explicitly working with relative packages, but you should rely on the compiled lib by just pointing to `'../../packages/react-cosmos-normalize-props-proxy'`